.pre-commit-config.yaml

```diff
@@ -8,6 +8,17 @@ repos:
     -   id: mixed-line-ending
     -   id: trailing-whitespace
 
+-   repo: https://github.com/ikamensh/flynt/
+    rev: '0.55'
+    hooks:
+    -   id: flynt
+        pass_filenames: false
+        args: [
+            '--line-length 120',
```

Comment:
@sphuber I think the reason it was behaving weirdly is that this should be `'--line-length', '120'`. At least locally for me, this fixes the issue.

Reply:
And we can remove the `pass_filenames: false` again - otherwise it'll either do nothing, or scan the entire code base.

Reply:
Ah yeah, I think either that or `'--line-length=12'` might also work

Reply:
Yep.. pretty unintuitive error message, I must say.